test.sh

```diff
@@ -27,9 +27,11 @@ npm test
 end test
 
 start cypress
-npm start & wait-on http://localhost:3000/
-npm run cypress:run
-echo 'NOTE: Server is still running.'
+if [ "$CI" != 'true' ]; then
```

Comment:
At line 10, `export CI=true`, so I'm not sure this will ever run? I'm not sure what the right solution is here: Before, we had tests, and the same test were run everywhere, but that's no longer the case... Maybe when testing locally with `test.sh` you just don't do cypress... if you get failures on CI, then you can run cypress locally with the UI open.

Reply:
I think I will leave a comment about this and remove the export at the top.  I think having the cypress run locally by default is a good thing.